Added script to see what needs review

noisecapella opened the pull request:
#### What are the relevant tickets?
Part of #74 
**Rebased** on #88

#### What's this PR do?
Adds a command `what needs review` which lists PRs needing review on all repos

#### How should this be manually tested?
Run `./bot_local.py what needs review`

annagav commented:
not sure how to run `bot_local.py`....

annagav commented:
```./bot_local.py what needs review
Traceback (most recent call last):
  File "./bot_local.py", line 36, in main
    repo_info = [repo_info for repo_info in repos_info if repo_info.name == project_name][0]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./bot_local.py", line 51, in <module>
    main()
  File "./bot_local.py", line 38, in main
    raise Exception("Unable to find channel with name {}".format(project_name))
Exception: Unable to find channel with name what```

noisecapella commented:
I made some changes in the PR I rebased on. I just rebased this PR, can you try again? The syntax should be, for example: `python3 bot.py bots what needs review` to run the command `what needs review` as if it was running in the `bots` channel
